core/src/main/PoliticalMap.kt

```diff
@@ -0,0 +1,72 @@
+class PoliticalMap private constructor(
+    private val countriesWithContinents: Map<String, String>,
+    private val borders: Map<String, Collection<String>>
+) {
```

Comment:
No se si representar paises y continentes directamente on strings. Preferiría usar o typedes (tipo c) si hay, o (mejor) hacer clases para cada cosa, tipo:

`class Country constructor(val name : String)`

Eso esta bueno porque ya tenemos una clase y eventualmente si lo necesitaoms podemos agregar atributos sin que el refactor sea demasiago grande (no haría que cambiar la signatura de todas las funsiones que usan Country). Lo mismo para continent.

Reply:
Yo pense lo mismo pero después pensé que si el country es solo un String no servía agregar complejidad. Aunque creo que hay typedefs, así que se podría hacer eso.

Lo otro que se podría hacer es que el politicalMap tenga los countries y continents como tipos paramétricos, y ahí si querés ponerle una clase después es fácil, es sólo cambiar donde lo inicializás (y donde se usa, pero bueno). 

Hay tres opciones: [typedef](https://www.google.com/search?client=firefox-b-d&q=kotlin+typedef), clase Country, o tipo paramétrico. Cuál te gusta más?

Reply:
Gustarme me gusta mas tipos param'etricos ;), pero no s'e si vale la pena en este caso porque no ve oque haya como un concepto de pa'is y paises de muchos tipos , etc. Si typedef funciona e nrealidad como 'using' de c++ (o sea se toma un tipo nuevo y no compila con el tipo del que es alias) entonces podriamos arrancar con typedesf hasta que sea necesario un refactor si queremos que tenga algun otro comportamiento o dato distinto de string. Que sea param'etrico el politicalMap es independiente, porque puede serlo aunque hicieramos clases para countries . Yo enotnces  esto lo dejaria asi y is queres hacemo suna task para reflexionar sobre si queremo so no tipos parametricoa ac'a.

Reply:
el typealias no hace nada más que hacer que sea más fácil escribir, no cambia nada en compilación.

Reply:
class Country entonces